reposerver/repository/repository.go

```diff
@@ -686,6 +687,8 @@ func (s *Service) runManifestGenAsync(ctx context.Context, repoRoot, commitSHA,
 		close(ch.responseCh)
 	}()
 
+	settings := operationSettings{sem: s.parallelismLimitSemaphore, noCache: q.NoCache, noRevisionCache: q.NoRevisionCache, allowConcurrent: q.ApplicationSource.AllowsConcurrentProcessing()}
```

Comment:
Instead of instantiating this settings object, what if we directly reference `q.NoCache` and `q.NoRevisionCache` below when we instantiate the cache?

Reply:
Great point, thanks, removed